how to connect cosmostation (#31)
bandprotocol/bandchain-docs: docs/staking/02-connect-wallet.md

Once you have clicked "Approve," you are now connected to Keplr. You can see the details of your account by clicking on the address button.
![account details](/img/staking/account_details.png)

## How to connect Cosmostation Wallet

### Prerequisites

Ensure that you have the Cosmostation Wallet extension installed on your browser.

> [download here](https://cosmostation.io/products/cosmostation_extension)

Reviewer: ```suggestion
[Download here](https://cosmostation.io/products/cosmostation_extension)
```

### 1. Add BandChain on Cosmostation

> if you don't see BandChain on your Cosmostation Extension. you can exsily activate it by click Add Chain button then search for Band and toggle swtich

Reviewer: ```suggestion
If you don't see BandChain on your Cosmostation Extension. you can easily add it by clicking on the Add Chain button then searching for Band and click on the toggle button to add BandChain.
```

![add bandchain to cosmostation](/img/staking/add-band-cosmostation.png)

### 2. Select Cosmostation Wallet options

> After you click the Connect Wallet button, a pop-up will appear showing wallet options.then select _Cosmostation_

Reviewer: ```suggestion
After you click the Connect Wallet button on CosmoScan website, a pop-up will appear showing wallet options. Then select _Cosmostation_
```
